sdk/ai/azure-ai-client/samples/evaluations/sample_evaluations.py

```diff
@@ -40,6 +40,9 @@
                     "azure_endpoint": "https://ai-anksingai0771286510468288.openai.azure.com/",
```

Comment:
I have noticed you have some personal information in the sample, it is publicly available, could be better to remove.